### PR TITLE
PMM-11500 reuse setup for BM

### PR DESCRIPTION
**Pull request by yurkovychv**
https://jira.percona.com/browse/PMM-11500

- [ ] Links to relevant pull requests

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-79265d7
client docker - perconalab/pmm-client-fb:PR-3035-79265d7
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-79265d7.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-79265d7&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-79265d7.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-b3f59f5
client docker - perconalab/pmm-client-fb:PR-3035-b3f59f5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-b3f59f5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-b3f59f5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-b3f59f5.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-5dee854
client docker - perconalab/pmm-client-fb:PR-3035-5dee854
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-5dee854.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-5dee854&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-5dee854.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-fa11bac
client docker - perconalab/pmm-client-fb:PR-3035-fa11bac
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-fa11bac.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-fa11bac&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-fa11bac.tar.gz

**Comment by JNKPercona** [comment]
Looks like there are outdated source branches.
 Please update them and restart the job
 https://github.com/percona/pmm-ui-tests/pull/505

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-88709d5
client docker - perconalab/pmm-client-fb:PR-3035-88709d5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-88709d5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-88709d5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-88709d5.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/12540/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-88709d5
client docker - perconalab/pmm-client-fb:PR-3035-88709d5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-88709d5.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-88709d5&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-88709d5.tar.gz

File: .github/workflows/ui-tests.yml [hunk 1]
```diff
@@ -138,7 +144,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ steps.artifact_name.outputs.result }}
-          path: tests/output
+          path: pmm-ui-tests/tests/output
```
Comment (vyarosh) [inline .github/workflows/ui-tests.yml:147]: Does it work properly without a dot in the relative path: `./pmm-ui-tests/tests/output` ?

Comment (yurkovychv) [reply .github/workflows/ui-tests.yml:147]: yes
it works fine this way

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-f219532
client docker - perconalab/pmm-client-fb:PR-3035-f219532
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-f219532.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-f219532&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-f219532.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/12575/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-3035-f219532
client docker - perconalab/pmm-client-fb:PR-3035-f219532
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-f219532.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-3035-f219532&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-3035-f219532.tar.gz

**Review by vyarosh — APPROVED** [review]

**Pull request merged**